### PR TITLE
Gamer Event Page

### PR DESCRIPTION
**Pull request by MarilenaRoque**
##Gamer Event Page

I have developed a website for a fictitious Gamer Event, focusing on applying a little bit of all the things that I have learned in this month on the Microverse Program. :rocket: 

It contains a mobile and desktop version for three different pages:
- Home
- About
- Tickets

1. Navbar
Added Links
Sticky position
Responsive Dropdown Menu

2. Main
Differents backgrounds using image and gradient

3. Main Program 
Used box-shadow to highlight the events
Set up a row display for the desktop version

4. Championship
Added the more button to hide some items in the mobile version
Set up responsiveness

5. Responsive Sponsors Section

6. Footer
Links for my social medias
Event Icon

7. About
Set up a hover effect for the images and headers
Set up responsiveness

8. Tickets
Added form
Radio buttons for lunch and days choose
Sheets using grid
Different display style for different screen size

### CODE REVIEW

**Comment by santiago-dev96** [comment]
# Pull Request under review by @santiago-rodrig (TSE)

File: index.html [hunk 1]
```diff
@@ -4,14 +4,293 @@
...
+
+    <link rel="shortcut icon" href="assets/images/small-logo.svg"> 
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="assets/font/stylesheet.css" type="text/css" />
+   
 
     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="./assets/styles/reset.css" type="text/css">
```
Comment (santiago-dev96) [inline index.html]: Bootstrap already includes a reset
```suggestion
```

**Review by santiago-dev96 — APPROVED** [review]
# You have achieved competence :rocket:

Please be aware that capstone projects should be reviewed by several TSEs, you can keep track of the project from your dashboard, once there are no more reviews to be made you can proceed to merge this PR.

Congratulations for doing a great job.

File: index.html (hunk 1 above)
Comment (MarilenaRoque) [reply index.html]: Got it. I will remove it before the merge. 
Thank you so much for your review. :smiley: :+1:

**Comment by MarilenaRoque** [comment]
I have fixed all the requirements for this project

- [x] Set up an opaque navbar
- [x] The mobile and desktop version works perfectly
- [x] Added h1 header for semantic HTML

Thank you so much for the reviews! :smile: :rocket:

**Pull request merged**